<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VMS管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>农户ID：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="purchaseIDLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>种类：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="purchaseKindcomBox">
                <property name="minimumSize">
                 <size>
                  <width>80</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>净重(Kg):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="purchaseWeightSpinBox">
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>价格(元):</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="purchasePriceDoubleSpinBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>50</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>60</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="minimumSize">
                 <size>
                  <width>35</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>时间:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="purchaseTimeDateEdit">
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                </property>
                <property name="dateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2000</year>
                  <month>1</month>
                  <day>1</day>
                 </datetime>
                </property>
                <property name="currentSection">
                 <enum>QDateTimeEdit::YearSection</enum>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="purchaseAddButton">
              <property name="text">
               <string>添加</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="purchaseTableView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="purchaseCancelButton">
              <property name="text">
               <string>撤销</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="purchaseRefreshButton">
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="purchaseCommitButton">
              <property name="text">
               <string>提交数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>姓名:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="farmerNameLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>编号:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="farmerIDLineEdit">
                <property name="maximumSize">
                 <size>
                  <width>80</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_20">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>性别:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="farmerSexCBox">
                <property name="maximumSize">
                 <size>
                  <width>35</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="editable">
                 <bool>false</bool>
                </property>
                <property name="maxVisibleItems">
                 <number>2</number>
                </property>
                <property name="maxCount">
                 <number>2</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>电话:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="farmerTelLineEdit">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>住址:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="farmerAddRLineEdit">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="farmerOkButton">
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="topMargin">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="farmerAddButton">
                <property name="text">
                 <string>录入</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="farmerSelectButton">
                <property name="text">
                 <string>查找</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="farmerAmendButton">
                <property name="text">
                 <string>修改</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="farmerDeleteButton">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="farmerStatusLabel">
                  <property name="minimumSize">
                   <size>
                    <width>15</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>300</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QPushButton" name="farmerCancelButton">
                <property name="text">
                 <string>撤销</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="farmerRefreshButton">
                <property name="text">
                 <string>刷新</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="farmerCommitButton">
                <property name="text">
                 <string>提交确认</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QStackedWidget" name="farmerStackedWidget">
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_9">
               <layout class="QHBoxLayout" name="horizontalLayout_26">
                <item>
                 <widget class="QTableView" name="farmerTableView"/>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_10">
               <layout class="QHBoxLayout" name="horizontalLayout_16">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_15">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>400</number>
                    </property>
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="maximumSize">
                       <size>
                        <width>60</width>
                        <height>16777215</height>
                       </size>
                      </property>
                      <property name="text">
                       <string>查找条件：</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_14">
                      <item>
                       <widget class="QCheckBox" name="farmerNameCheckBox">
                        <property name="text">
                         <string>姓名</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QCheckBox" name="farmerIDCheckBox">
                        <property name="text">
                         <string>编号</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QCheckBox" name="farmerSexCheckBox">
                        <property name="text">
                         <string>性别</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QCheckBox" name="farmerAddrCheckBox">
                        <property name="text">
                         <string>住址</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QTableView" name="farmerSelectTableView"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="filemenu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="purchaseDataAction"/>
    <addaction name="farmerDataAction"/>
    <addaction name="separator"/>
    <addaction name="regAction"/>
    <addaction name="amendAction"/>
    <addaction name="separator"/>
    <addaction name="exitaction"/>
   </widget>
   <widget class="QMenu" name="helpmenu">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="currrentUserAction"/>
   </widget>
   <addaction name="filemenu"/>
   <addaction name="helpmenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="exitaction">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="farmerDataAction">
   <property name="text">
    <string>农户资料</string>
   </property>
  </action>
  <action name="purchaseDataAction">
   <property name="text">
    <string>收购数据</string>
   </property>
   <property name="toolTip">
    <string>收购数据</string>
   </property>
  </action>
  <action name="regAction">
   <property name="text">
    <string>注册用户</string>
   </property>
   <property name="toolTip">
    <string>注册用户</string>
   </property>
  </action>
  <action name="amendAction">
   <property name="text">
    <string>密码修改</string>
   </property>
   <property name="toolTip">
    <string>密码修改</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="currrentUserAction">
   <property name="text">
    <string>当前用户</string>
   </property>
   <property name="toolTip">
    <string>当前用户</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
